scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
+    box-shadow: $crm-form-control-shadow;
+    border: 1px solid $crm-darker-grayblue;
+    line-height: 1em;
+    padding: 8px 15px;
+    margin-right: -5px;
+    min-width: 180px;
+
+
+    &.active {
+      background-color: $gray-lighter;
+    }
+
+    &:first-child {
+      border-radius: 2px 0 0 2px;
+    }
+
+    &:last-child {
+      border-radius: 0 2px 2px 0;
+    }
+
+    input {
+      margin-right: 10px;
```

Comment:
It doesn't has in the example. but it has margin-left:-20px; 
and it also should has the wrapping ".radio" to take this margin-left:-20px; so i have to change the markup to match this 

I also tried to change the markup from inspect element and i found that you override the margin-left:-20px  in shoreditsh theme

![screen shot 2017-04-06 at 11 45 17 am](https://cloud.githubusercontent.com/assets/2423218/24748049/be4b136c-1abe-11e7-940f-bd21d8adf1b5.png)